src/test/java/net/snowflake/client/jdbc/ResultSet0IT.java

```diff
@@ -23,79 +23,78 @@ public class ResultSet0IT extends BaseJDBCTest {
 
   public Connection init(int injectSocketTimeout) throws SQLException {
     Connection connection = BaseJDBCTest.getConnection(injectSocketTimeout);
```

Comment:
should be in a try

Reply:
This method returns the connection instance. Should not be closed.